src/Presentation/Views/Registration/Registration.tsx

```diff
@@ -1,11 +1,30 @@
+import useViewModel from './RegistrationViewModel';
 
 function Registration() {
 
+    const { register, onChangeEmail, onChangePassword } = useViewModel();
+
+
     return (
         <div>
             <h1>Registration Page</h1>
+            <h3>Register to use Dodo</h3>
+            <input 
+                placeholder="Email..." 
+                onChange={(event) => onChangeEmail(event.target.value)}
+            />
+
+            <input 
+                placeholder="Password..."
+                type="password"
+                onChange={(event) => onChangePassword(event.target.value)}
```

Comment:
just curious nit, is there any way to remove the `(event) => ` from this? In kotlin, if you have a single variable like this it will just become a keyword, `it`, for example, in kotlin, it would let you simplify the code to

`onChange = { onChangePassword(it) }`

Reply:
Updated